src/GitTools.IssueTrackers/Context/IssueTrackerContext.cs

```diff
@@ -1,29 +0,0 @@
-﻿namespace GitTools.IssueTrackers
-{
-    public class IssueTrackerContext : Disposable, IIssueTrackerContext
-    {
-        public IssueTrackerContext()
```

Comment:
It's the context of the issue tracker at that moment. But feel free to replace it with info. The reason I implement IDisposable is that you can use scopes for contexts and this always allows you to clean up. For example, dynamic repository contexts might have temporary files that need cleanup.

If everything context implements IDisposable, you can do a cleanup by only disposing the top-level context (which is the app-context).